README.md

```diff
@@ -39,6 +40,13 @@ Also this repository contains shared packages that simplify development and test
 Feel free to reach out to us with the ones that'd be the most useful to have
 here by filing a [Github issue][github-issues]!
 
+### Styles
+
+If you are using field editors in a project, for them to render
```

Comment:
Tiny nit to make it more compact maybe:

---

To achieve the same field editor look as in the Contentful UI, you need to include [Contentful's Forma 36 styles](https://www.npmjs.com/package/@contentful/forma-36-react-components) in your project.

```js
import '@contentful/forma-36-react-components/dist/styles.css';
```